tests/testsuite/cargo_information/pick_msrv_compatible_package/stderr.log

```diff
@@ -0,0 +1,3 @@
+    Updating `dummy-registry` index
+ Downloading crates ...
+  Downloaded my-package v0.2.0+my-package (registry `dummy-registry`)
```

Comment:
I think this is a bug. We should pick the MSRV-compatible version: `0.1.1+my-package`